zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -629,6 +646,11 @@ impl Parameters {
     pub fn disable_pow(&self) -> bool {
         self.disable_pow
     }
+
+    /// Returns the halving interval for this network
+    pub fn halving_interval(&self) -> Height {
```

Comment:
This should be used where `PRE_BLOSSOM_HALVING_INTERVAL` or `POST_BLOSSOM_HALVING_INTERVAL` are currently being used.